### DISCUSSION TITLE
Security...

### DISCUSSION THREAD

**Discussion by 4iMh9CawHdyu** [linux-surface/linux-surface#1264] [Q&A] [answered]
Just a quick question, don't know if it'll be a silly ignorant one but here comes: If I use a linux surface kernel, say debian, how does it update for patches and security? Thanks in advance

**Comment by qzed** [comment, marked as answer]
What specifically do you mean? We try to keep our kernel packages up-to-date with the latest upstream sources. For Ubuntu/Debian packages, those come from the Ubuntu mainline repo. As for update delivery: We've set up a repo for Debian/Ubuntu (as well as Arch Linux and Fedora), which directly works with your package manager when set up. So when you use that to update, the kernel packages get updated alongside the rest of the system.

**Reply by 4iMh9CawHdyu** [reply]
Ok. Just Checking. Great. Thank you very much.
